python/tchannel/tornado/tchannel.py

```diff
@@ -51,6 +56,12 @@ class State(IntEnum):
 class TChannel(object):
     """Manages inbound and outbound connections to various hosts."""
 
+    _SCHEMES = {
+        'raw': scheme.RawArgScheme,
+        'json': scheme.JsonArgScheme,
```

Comment:
Actually I remember I had it in my retry patch. But it is quite out of date right now.